JM/CF/Scripts/3_Game/CommunityFramework/Timer/CF_DelayCall.c

```diff
@@ -0,0 +1,69 @@
+class CF_DelayCall : CF_TimerBase
```

Comment:
> I am not sure what the vision is for DelayCall ... the name suggests delaying ONE function call to a later time ... aka the same that ScriptCallQueue already does.

Yeah. According to the source that I've outlined in private messages to you, ScriptCallQueue has issues when a large wait time has been inputted. 

> Do you maybe mean RepatedCall/ReapeatableCall since we need to store the result of CF_DelayCall::Create() somewhere already?

FYI, that's what `CF_Timer` is doing already